content/docs/join_us/recruitment_FAQs.md

```diff
@@ -160,24 +275,40 @@ Using existing libraries that implement algorithms you are using is fine (and go
 1. the underlying algorithms or methods your code is implementing or using, and
 1. the research domain or question for which you are applying them.
 
-### Can you give any more detail on what the panel expect from the algorithm which will be presented - does it need to meet any certain criteria, for example have a certain level of complexity?
+We will not evaluate you on the complexity of the algorithm, but in your understanding of its implementation, awareness of its potential drawbacks, and its overall appropriateness to solve the problem you are trying to address.
+
+You can write new code specifically for the interview if you so choose, but existing examples of work are encouraged, as long as the code was substantially written by you.
+You should be prepared to answer questions about both the code and the research challenge it addresses or the algorithm it demonstrates.
+Any source code shared for the interview will be treated in the strictest of confidence.
 
-In the first interview we expect you to demonstrate your ability to write code for addressing a data science task and/or to use good research software engineering practices for ensuring research reproducibility.
-This could be done by presenting for instance a data analysis script written by you, where you would describe us the implementation and answer some general questions.
-The adopted algorithms can be ones available in existing software libraries, integrated for instance in a data analysis workflow, but they can also be written by you if you prefer.
+### What sort of code have previous candidates presented?
 
-We will not evaluate you on the complexity of the algorithm, but in your understanding of its implementation, awareness of its potential downfall and its overall appropriateness to solve the problem you are trying to address.
+We collected several examples of code that have been presented by current team members in their interviews.
+Some are scripts / notebooks which were put together specifically to demonstrate a particular technique for the interview:
 
-### Will I be only explaining the algorithm / analysis / software, or running it and discussing the results?
+- A Jupyter notebook with a data analysis pipeline to solve an old Kaggle competition
+- A notebook (and module0 demonstrating inference and analysis with a hidden Markov model using TensorFlow Probability, applied to synthetic healthcare data
```

Comment:
For all the examples here that aren't ours, I've asked the people who submitted them for explicit consent. There are more which I haven't yet added.

I'd like to have more examples of not-libraries, but the 2 other people who submitted this are off work atm 😅 We can add more later but for now I think this is a decent start.

Reply:
I'm happy to include my example (though I'd have to put it on GitHub first) - it wasn't a library, just a function for analysing some data.

Reply:
Here's a link - https://github.com/IFenton/Sediment-age-depth. When I get back, I'll add the data so it can actually be run, but that will need a bit of tidying / checking permissions